src/js/callCensus.js

```diff
@@ -1,32 +1,37 @@
-var $ = require('jquery');
+require( 'es6-promise' ).polyfill();
+var jsonP = require( 'jsonp-p' ).default;
+var DT = require( './dom-tools' );
 
-module.exports = function(address, rural, cb) {
-  $.ajax({
-    url: 'https://geocoding.geo.census.gov/geocoder/locations/onelineaddress?',
-    dataType: 'jsonp',
-    data: {
-      address: address,
-      benchmark: '4',
-      format: 'jsonp'
-    },
-    success: function(data) {
-      callbacks.censusAPI(data, rural);
+module.exports = function( address, rural, cb ) {
+  var url = 'https://geocoding.geo.census.gov/geocoder/locations/onelineaddress?';
+  url += 'address=' + address;
+  url += '&benchmark=4'
+  url += '&format=jsonp';
+
+  jsonP( url ).promise
+  .then( function( data ) {
+      window.callbacks.censusAPI( data, rural );
     }
-  })
-  .fail(function(jqXHR, textStatus) {
-    if (jqXHR.status !== 200) {
-      $('#processErrorDesc').append('<li>' + address + '</li>');
-      $('#processError').removeClass('hide');
+  )
+  .catch( function( error ) {
+    if ( error ) {
+      var addressElement = DT.createEl( '<li>' + address + '</li>' )
+
+      DT.addEl( DT.getEl( '#processErrorDesc' ), addressElement );
```

Comment:
When this happens I'm seeing

![processingerror](https://user-images.githubusercontent.com/2932572/36260957-df5923e0-1230-11e8-9b6a-7656a52db0ed.png)

Notice the `[object DocumentFragment]` instead of the address. Also notice the loading icon still there, near the counts.

When this happens on the current production site you'll see

![processingerrorgood](https://user-images.githubusercontent.com/2932572/36261033-112acee6-1231-11e8-8128-4972e9b6b9bc.png)

The correct address is rendered and the count completes (no loading icon). This seems to be blocking the completion of the process ... if you try a [larger file](https://github.com/cfpb/rural-or-underserved-test/blob/bbf2d0c1dcbf0283e2f6e85bf7d0355552c3d697/test/noCty-Blk_formatted.csv) it's easier to see.